slither/solc_parsing/declarations/function.py

```diff
@@ -1332,9 +1332,13 @@ def _parse_params(self, params: Dict):
         else:
             params = params[self.get_children("children")]
 
+        next_unnamed_param_id = 0
         for param in params:
             assert param[self.get_key()] == "VariableDeclaration"
             local_var = self._add_param(param)
+            if self.slither_parser.generates_certik_ir and local_var.underlying_variable.name == "":
+                local_var.underlying_variable.name = f"ANON_IN_{next_unnamed_param_id}"
+                next_unnamed_param_id += 1
```

Comment:
Why don't we use their actual parameter index? So, their number will be more meaningful and no need for `next_unnamed_param_id` to make them unique.